### ISSUE TITLE
New ID request: Reflex anoxic seizure

### ISSUE THREAD

**Issue by evans-dan** [obophenotype/human-phenotype-ontology#4232]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label

Reflex anoxic seizure

### 2. Synonyms
Reflex asystolic syncope, reflex anoxic seizures, RAS

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

A loss of consciousness followed by stiffening and brief clonic movements affecting some or all limbs, often misinterpreted as an epileptic seizure.

PMID: 23814085 
doi: 10.1136/archdischild-2012-303133

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))

HP:0001279

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)

OMIM:607578
